packages/orchestration/src/examples/continuing-invitation.contract.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @file This contract demonstrates the continuing invitation pattern with async
+ *   flows.
+ *
+ *   The primary offer result is a power for invitation makers that can perform
+ *   actions with an ICA account.
```

Comment:
Our other examples describe what the contract does, not a technical pattern. Struggling to come up with a good name - maybe something like `Staking (Automations | Flows | Extended | Extras)`

Reply:
yeah I was noodling on this too. "Continue" does seem to be a feature though, that you get a wrapper for the Cosmos account that provides operations through smart-wallet.

`ContinuingCosmos`?  `WalletCosmosAccount`?